### PR TITLE
Redesign attachment and comment sections

### LINKED ISSUE #308: Process versions in process edit have redundant information.
It should look like this:
version-history_attachments

### LINKED ISSUE #331: numbers attached to nodes in test view and counts view can look better
it should look like this:
counts3

bg color: #4d4d4d
text color: #ccc
font size on default zoom: 16px
padding on default zoom: 8px

this is how they look now:
Clipboard - November 8, 2019 12_45 PM

### LINKED ISSUE #346: Layout action with counts enabled
It looks like the "layout" action does not user wider node dimensions for calculation when "counts" are enabled.

### LINKED ISSUE #355: Remove edit > "Duplicate" button
This action predates ^C ^V and can be removed.

### LINKED ISSUE #384: comments need some padding
Now it looks like this:
2019-11-18 09_17_05-Nussknacker

- Comments' text width should match the width of hairline splitter above.
- Also, every comment should have 1px top margin to slightly separate it from the date
- In the comment textarea we need to add 4px padding.

### LINKED ISSUE #385: better looking attachments section
should look like this:
version-history_attachments

### PR DESCRIPTION
**Pull request by PPiedel**
This resolves #308 
This resolves #384
This resolves #385
This resolves #355
This resolves #331
This resolves #346

### CODE REVIEW

File: ui/client/components/ProcessAttachments.js [hunk 1]
```diff
@@ -40,20 +39,22 @@ export class ProcessAttachments_ extends React.Component {
...
                       onClick={this.downloadAttachment.bind(this, attachment.processId, attachment.processVersionId, attachment.id)}
                   />
                 </div>
-                <div className="header">
-                  <span className="label label-info">{attachment.user}</span>
-                  <span title={DateUtils.formatAbsolutely(attachment.createDate)} className="date">{DateUtils.formatRelatively(attachment.createDate)}</span>
-                  <p>{ProcessUtils.processDisplayName(attachment.processId, attachment.processVersionId)}</p>
+                <div className={"attachment-details"}>
+                  <div className="header">
+                  <span title={DateUtils.formatAbsolutely(attachment.createDate)} className="date">
```
Comment (lciolecki) [inline ui/client/components/ProcessAttachments.js]: What do you think about new component to presenting Date?

**Comment by witekw** [comment]
image

image

date + version + user looks different in two places above, is it on purpose?

**Comment by witekw** [comment]
I like new "counts", it solves #346

**Comment by fi-** [comment]
> image
> 
> image
> 
> date + version + user looks different in two places above, is it on purpose?

Good point, we should do the comments the same as attachments

File: ui/client/components/ProcessAttachments.js (hunk 1 above)
Comment (PPiedel) [reply ui/client/components/ProcessAttachments.js]: Done

**Pull request merged**